lib/collections/addon/provider/moderation/template.hbs

```diff
@@ -1,2 +1,46 @@
-{{!-- Moderation Parent route content --}}
-{{outlet}}
\ No newline at end of file
+<OsfLayout @backgroundClass={{local-class 'content-background'}} as |layout|>
+    <layout.heading local-class='heading'>
+        <div local-class='clamped-width-container'>
+            <h1>
+                {{t 'collections.moderation.header'}}
+            </h1>
+        </div>
+    </layout.heading>
+    <layout.main local-class='main'>
+        <BsNav local-class='clamped-width-container' @type='tabs' as |nav|>
+            {{#let 'collections.provider.moderation.all' as |pendingRoute|}}
+                <nav.item @active={{eq this.target.currentRouteName pendingRoute}}>
+                    <OsfLink
```

Comment:
Same here for aria-labels and focusability. Other than that, these changes look good!

Reply:
This link doesn't need an aria-label either, since it will always have visible text content. As for focusability, same thing. I've tried it out and it doesn't have any nested interactable elements

Reply:
Ok, that sounds good